### PR TITLE
Make an account filter for abilities enabled by default

### PR DESCRIPTION
**Pull request by kkosiorowska**
This PR makes an account filter for abilities enabled by default.

_Originally posted in https://github.com/tallyhowallet/extension/pull/2951#pullrequestreview-1277341370_

Latest build: [extension-builds-2954](https://github.com/tallyhowallet/extension/suites/10807599607/artifacts/543663630) (as of Mon, 06 Feb 2023 16:13:18 GMT).

### CODE REVIEW

File: background/main.ts [hunk 1]
```diff
@@ -1534,7 +1534,9 @@ export default class Main extends BaseService<never> {
       this.store.dispatch(addAbilities(newAbilities))
     })
     this.abilitiesService.emitter.on("newAccount", async (address) => {
-      this.store.dispatch(addAccount(address))
+      if (isEnabled(FeatureFlags.SUPPORT_ABILITIES)) {
```
Comment (0xDaedalus) [inline background/main.ts:1537]: 📈

**Review by 0xDaedalus — APPROVED** [review]
🎉 🎉 🎉

**Review by 0xDaedalus — DISMISSED** [review]
Code looks great and manual testing works - but this seems like an excellent candidate for a regression test.

**Comment by kkosiorowska** [comment]
> Code looks great and manual testing works - but this seems like an excellent candidate for a regression test.

I did a regression test manually. It seems that everything is working properly. If you have any suggestions on what tests we should still add to the code let me know.  @0xDaedalus @jagodarybacka

**Review by jagodarybacka — DISMISSED** [review]
Hmm, I'm not sure what test you had in mind @0xDaedalus 🤔 for "newAccount" event? 
LGTM ✅

**Review by jagodarybacka — APPROVED** [review]
✅ Let's merge and include tests in next PR if needed

**Comment by puppycodes** [comment]
Tests are needed for this

**Pull request merged**